app/app.go

```diff
@@ -142,7 +142,7 @@ func init() {
 		panic(err)
 	}
 
-	DefaultNodeHome = filepath.Join(userHomeDir, ".sifenoded")
+	DefaultNodeHome = filepath.Join(userHomeDir, ".sifnoded")
```

Comment:
@iverc @marbar3778 - not sure why we're doing this anyway, as `DefaultNodeHome` is already set on line 99...